### ISSUE TITLE
Initialize Rustdocs

### ISSUE THREAD

**Issue by corysabol** [NiceDuckGames/Jovia#20]
**Describe the solution you'd like**
The bindings need to have sufficient documentation. Fortunately cargo provides us with a nice wrapper around rustdoc that allows for the generation of decent API documentation from the source code and documentation comments. 

We may need to look into where to host this documentation down the line, such as on docs.rs. However, I think it would also be sufficient to simply host the docs in the repo wiki if it is easy enough to do so with the generated docs.

**Comment by corysabol** [comment]
I agree, the wiki format is kind of annoying. The issue I see happening with it is that the wiki won't really facilitate the the format that rustdocs generates well without some leg work. And I'm not too into that.

**Comment by wardensdev** [comment]
> without some leg work.
Yeah, everybody hates leg day.

Maybe there's some kind of tool that can generate a more suitable format for a wiki and just use both? Either way docs.rs would be the starting point i think.
